scripts/validation/run_predictive_success_campaign.py

```
@@ -257,41 +310,84 @@ def main() -> int:
+                )
+                results.append(
+                    {
+                        "checkpoint": checkpoint,
+                        "variant": name,
+                        "config": cfg,
+                        "hard": hard_res.__dict__,
+                        "global": global_res.__dict__,
+                        "ranking_key": list(_rank_key(hard_res, global_res)),
+                    }
+                )
+    except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)